code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -307,7 +307,7 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/_comma.ogg',
 "inspector" = 'sound/vox_fem/inspector.ogg',
 "interchange" = 'sound/vox_fem/interchange.ogg',
 "intruder" = 'sound/vox_fem/intruder.ogg',
-"invallid" = 'sound/vox_fem/invallid.ogg',
+"invalid" = 'sound/vox_fem/invalid.ogg',
```

Comment:
i thought this shit automatically figured out vox files to play automatically, wack. cool that you picked this up tho

Reply:
is it still invallid?

Reply:
kek